### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
<em>Describe the steps with bullet points. Is there any external link that can be used to better test it or an example?</em> 

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

**Review by emersonlaurentino — APPROVED** [review]

**Review by Gmantiqueira — APPROVED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ vtex login {account}
 vtex install vtex.admin-search@1.x vtex.admin-cms@0.x vtex.graphql-gateway@0.x vtex.store-sitemap@2.x
 ```
 
+>ℹ️ *Make sure VTEX Intelligent Search is installed and set correctly. Visit our (help center)[https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/3qgT47zY08biLP3d5os3DG] for more info*
+
 3. [Create a new repository from this template](https://github.com/vtex-sites/storecomponents.store) inside the [`vtex-sites` organization.](https://github.com/vtex) with the name `{account}.store`.
```
Comment (gris) [inline docs/getting-started.md:44]:
```suggestion
3. [Create a new repository from this template](https://github.com/vtex-sites/storecomponents.store) inside the [`vtex-sites` organization.](https://github.com/vtex-sites) with the name `{account}.store`.
```

Comment (gris) [reply docs/getting-started.md:44]: Not related to your PR, but adding here anyway.

**Review by gris — APPROVED** [review]

**Review by klzns — APPROVED** [review]

File: docs/getting-started.md [hunk 2]
```diff
@@ -39,7 +39,9 @@ vtex login {account}
 vtex install vtex.admin-search@1.x vtex.admin-cms@0.x vtex.graphql-gateway@0.x vtex.store-sitemap@2.x
 ```
 
-3. [Create a new repository from this template](https://github.com/vtex-sites/storecomponents.store) inside the [`vtex-sites` organization.](https://github.com/vtex) with the name `{account}.store`.
+>ℹ️ *Make sure VTEX Intelligent Search is installed and set correctly. Visit our (help center)[https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/3qgT47zY08biLP3d5os3DG] for more info*
```
Comment (lucasecdb) [inline docs/getting-started.md:42]:
```suggestion
>ℹ️ *Make sure VTEX Intelligent Search is installed and set correctly. Visit our [help center](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/3qgT47zY08biLP3d5os3DG) for more info*
```

Comment (carolinamenezes) [inline docs/getting-started.md:42]:
```suggestion
>ℹ️ *Before proceeding, make sure that the [VTEX Intelligent Search](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/3qgT47zY08biLP3d5os3DG) app (`vtex.search-resolver@1.x`) is installed in your VTEX account and that [indexing](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/6wKQgKmu2FT6084BJT7z5V) has started. Check [our docs](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/3qgT47zY08biLP3d5os3DG) for more info.*
```

Comment (carolinamenezes) [reply docs/getting-started.md:42]: Boa! Deixei mais específico para a doc que redireciona para o IO Docs não confundir ngm.

**Pull request merged**